charts/hpe-csi-driver/v1.3.0/Chart.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+appVersion: 1.3.0
+description: A Helm chart for installing the HPE CSI Driver for Kubernetes
+icon: https://raw.githubusercontent.com/hpe-storage/co-deployments/master/docs/assets/hpedev.png
```

Comment:
Can you make a local copy of the logo in hpe-csi-driver/ and reference it here as file://../hpedev.png? This is so that airgap setups can use it. Thanks